### DISCUSSION TITLE
Azure Function vs App Service

### DISCUSSION THREAD

**Discussion by serottez** [Azure/ipam#320] [Q&A]
I am considering both the Azure Function and App Service deployment model for running the IPAM Container application in an App Service Plan. Can you detail the pros and cons of each implementation? Thank you!

**Comment by DCMattyG** [comment]
Hi @serottez, happy to help provide some guidance for you here!

Honestly, there isn't a distinct advantage in one vs. the other. They run effectively the same code, however some organizations prefer App Service, whereas other prefer to leverage Functions instead. App Service might be better for larger implementations as you can scale App Services up fairly large these days, but for most folks I would simply say to pick the one you prefer the most and go with that.

Hope that helped. Happy to dive into additional details if you'd like!

**Comment by serottez** [comment]
Thank you, Matt!
